config/initializers/geocoder.rb

```diff
@@ -37,3 +37,15 @@
 else
   Geocoder.configure(defaults)
 end
+
+module BingOverrideURLParams
+  def query_url_params(query)
+    { c: 'en-gb' }.merge(super)
```

Comment:
This gives precendence to the whatever the parent `query_url_params` defines ?

Do we want this to be the other way round, ie `super.merge(c: 'en-gb')`